examples/pymc3_howto/lasso_block_update.ipynb

```diff
@@ -20,6 +20,10 @@
    "outputs": [],
```

Comment:
<p>best to remove.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>